client/src/review-components/HoverStars.jsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import ReactDOM from 'react-dom/client';
+
+//to use in any component, pass the rating into props as rating='<Float>'
+
+function HoverStars(props) {
+  const [starRating, setStarRating] = useState(0);
+  const [starFill, setStarFill] = useState(0);
+
+  function handleClick(e) {
+    props.selectRating(e);
+    setStarRating(e.target.value);
+  }
+
+  return (
+    <div className='hoverStar-rating'>
+      {[...Array(5)].map((star, index) => {
+        index += 1;
```

Comment:
Maybe add a note for why you're accumulating in the index? It already goes up by 1 for each element